<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Review</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="viewTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>821</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="delError">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Error label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
